README.md

```diff
@@ -88,9 +89,56 @@ to tools provided by the `reset`. The `obs_msgs` returned from the environment a
 general messages that are observations. The `reward` is a scalar value. The `done` is a boolean value. The `truncated`
 is a boolean value.
 
-Let's see a complete example for building an environment.
+## Functional Environments
 
-### Environment subclass and state
+The easiest way to create an environment is using the functional interface, which just uses functions and decorators to define environments. First, let's define what the environment looks like by defining its `start` function:
+
+```py
+from aviary import fenv
```

Comment:
What do you think of matching `torch` here:

```python
from aviary import fenv as F

@F.start()
...
```

Reply:
How about we use jquery - $ :) 

I like fenv. No need to conflict with torch import

Reply:
Ok - I guess we could use `A`? wdyt?

Reply:
Yeah a conflict is annoying, agreed and nice point.

I was just trying to think about how to get a more intuitive name than "fenv", which only makes sense once one understand everything

Reply:
I think `A` is decent, but maybe we YAGNI this too